actionpack/lib/action_dispatch/log_subscriber.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module ActionDispatch
+  class LogSubscriber < ActiveSupport::LogSubscriber
+    def redirect(event)
+      payload = event.payload
+      status = payload[:status]
+
+      message = +"Completed #{status} #{Rack::Utils::HTTP_STATUS_CODES[status]} in #{event.duration.round}ms"
+      message << "\n\n" if defined?(Rails.env) && Rails.env.development?
+
+      info "Redirected to #{event.payload[:location]}"
+      info message
```

Comment:
You want to use an `info do` block like other LogSubscriber classes, otherwise you always build the string regarless of the log level. Not a huge deal, but it's mostly for consistency.